jest.setup.js

```diff
@@ -9,3 +9,11 @@ jest.mock('next/image', () => ({
     return props.src;
   },
 }));
+
+window.matchMedia = jest.fn().mockImplementation(query => ({
+  matches: false,
+  media: query,
+  onchange: null,
+  addListener: jest.fn(),
+  removeListener: jest.fn(),
+}));
```

Comment:
Could you share more about the purpose of this? 🙇

Reply:
`react-slick` faces this issue with jest while running tests, their test setup shows how to resolve it: https://github.com/akiran/react-slick/blob/master/test-setup.js#L9

Reply:
Thanks!!! :)

Reply:
Should we add a small comment on line 12, to remember that? :) 
For example `Fix jest issue with the react-slick component`?

Reply:
sure, done in 664dcd34571dbbbd6d26d745024c495892f75226